src/control/provider/system/distribution.go

```diff
@@ -187,3 +190,21 @@ func getDistribution(open openFunc) Distribution {
 func GetDistribution() Distribution {
 	return getDistribution(os.Open)
 }
+
+// GetLspciPath returns binary path based on distribution.
+func GetLspciPath() (string, error) {
+	distro := GetDistribution()
+
+	switch distro.ID {
+	case "opensuse-leap", "opensuse", "sles":
+		envPath, ok := os.LookupEnv("PATH")
+		if !ok {
+			return "", errors.New("couldn't find PATH env")
+		}
+		if err := os.Setenv("PATH", common.AppendToPathEnv(envPath, "/sbin")); err != nil {
```

Comment:
Could be more safe to add this path at the beginning of the PATH environment variable.

Reply:
@mjmac @kjacque any thoughts on this? I will change if there is a consensus.

Reply:
Often I've seen the value added to the front of the PATH to ensure the right binary is picked up. That said, I don't have strong feelings about it, particularly in this case.

Reply:
putting /sbin to the front of PATH (if it isn't already there) can have unintended side-effects. here we only want to make sure that we find the pci tools, so having it at the end should be fine and is safer.